src/swift/Data.swift

```diff
@@ -10,43 +10,55 @@
 //
 //===----------------------------------------------------------------------===//
 
-public struct DispatchData : RandomAccessCollection, _ObjectiveCBridgeable {
+import CDispatch
+
+public struct DispatchData : RandomAccessCollection {
 	public typealias Iterator = DispatchDataIterator
 	public typealias Index = Int
 	public typealias Indices = DefaultRandomAccessIndices<DispatchData>
 
 	public static let empty: DispatchData = DispatchData(data: _swift_dispatch_data_empty())
 
+#if false /* FIXME: dragging in _TMBO (Objective-C) */
 	public enum Deallocator {
 		/// Use `free`
 		case free
 
+#if HAVE_MACH
```

Comment:
This isn't a mach specific thing from my understanding for dispatch: it probably could be just like the backing for Data;
https://github.com/apple/swift-corelibs-foundation/blob/master/Foundation/Data.swift#L19